GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2835,7 +2835,21 @@ private void GoToMergeBase()
                 revisions
             };
 
-            string mergeBaseCommitId = UICommands.Module.GitExecutable.GetOutput(args).TrimEnd('\n');
+            ExecutionResult result = UICommands.Module.GitExecutable.Execute(args, throwOnErrorExit: false);
+
+            // Not related histories
+            if (result.ExitCode == 1)
```

Comment:
Consider adding a `const` and a link to the git doc as in `GitModule.GetGitSetting`.